bokeh/plotting/helpers.py

```diff
@@ -346,6 +346,8 @@ def _get_range(range_input):
         return FactorRange(factors=sorted(list(range_input.groups.keys())))
     if isinstance(range_input, Range):
         return range_input
+    if pd and isinstance(range_input, pd.Series):
+        range_input = range_input.values
```

Comment:
We probably need to validate that the length of the series is exactly two and raise an informative `ValueError` if not

Reply:
Actually I just expose the underlying numpy array, and let this be handled by the next 'if' statement. I've added tests that reflect this.